modules/geth/charts.go

```diff
@@ -0,0 +1,182 @@
+package geth
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dims   = module.Dims
+	Dim    = module.Dim
+)
+
+var charts = Charts{
+	chartAncientChainData.Copy(),
+	chartChaidataDisk.Copy(),
+	chartTxPoolPending.Copy(),
+	chartNewBlocksCreation.Copy(),
+	chartTxPoolQueued.Copy(),
+	chartP2PNetwork.Copy(),
+	chartP2PNetworkDetails.Copy(),
+	chartNumberOfPeers.Copy(),
+	chartRpcInformation.Copy(),
+	chartAncientChainDataRate.Copy(),
+	chartChaidataDiskRate.Copy()
+}
+
+var (
+
+	chartAncientChainDataRate = Chart{
+		ID:    "chaindata_ancient_rate",
+		Title: "Ancient Chaindata rate",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.eth_db_chaindata_ancient_io_rate",
+		Dims: Dims{
+			{ID:ethDbChainDataAncientRread , Name: "reads", Algo: "incremental"},
+			{ID: ethDbChainDataAncientWrite, Name: "writes", Mul: -1, Algo: "incremental",},
+		},
+		
+	}
+
+	chartAncientChainData = Chart{
+		ID:    "chaindata_ancient",
+		Title: "Ancient Chaindata",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.eth_db_chaindata_ancient_io",
+		Dims: Dims{
+			{ID:ethDbChainDataAncientRread , Name: "reads"},
+			{ID: ethDbChainDataAncientWrite, Name: "writes", Mul: -1},
+		},
+		
+	}
+	chartChaidataDiskRate = Chart{
+		ID:    "chaindata_disk_date",
+		Title: "On disk Chaindata rate",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.eth_db_chaindata_disk_io_rate",
+		Dims: Dims{
+			{ID:ethDbChaindataDiskRead , Name: "reads", Algo: "incremental"},
+			{ID: ethDbChainDataDiskWrite, Name: "writes", Mul: -1, Algo: "incremental"},
+		},
+	}
+	chartChaidataDisk = Chart{
+		ID:    "chaindata_disk",
+		Title: "Chaindata on disk",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.eth_db_chaindata_disk_io",
+		Dims: Dims{
+			{ID:ethDbChaindataDiskRead , Name: "reads"},
+			{ID: ethDbChainDataDiskWrite, Name: "writes", Mul: -1},
+		},
+	}
+	chartNewBlocksCreation = Chart{
+		ID: "chaindata_block_rate", 
+		Title: "Block creation rate",
+		Units: "blocks",
+		Fam: "block_processing", 
+		Ctx: "geth.block_processing",
+		Dims: Dims{
+			{ID: chain_head_block, Name: "new blocks", Algo: "incremental"},
+		},
+	}
+	chartTxPoolPending = Chart{
+		ID: "txpoolpending",
+		Title: "Pending Transaction Pool",
+		Units: "transactions", 
+		Fam: "tx_pool",
+		Ctx: "geth.tx_pool_pending", 
+		Dims: Dims{
+			{ID: txPoolInvalid, Name: "Invalid transaction pool"},
+			{ID: txPoolPending, Name: "Pending transaction pool"},
+			{ID: txPoolLocal, Name: "Local transaction pool"},
+			{ID: txPoolPendingDiscard, Name: "Pending discard transaction pool"},
+			{ID: txPoolNofunds, Name: "Pool of transactions with no funds" },
+			{ID: txPoolPendingRatelimit, Name: "Pending transaction pool ratelimit"},
+			{ID: txPoolPendingReplace, Name: "Pending transaction pool to replace"},
```

Comment:
Are you sure about adding `transaction pool` to every dimension name?

We have `Transaction Pool` in the title + `transactions` as units. Why don't just `invalid`, `pending`, etc.?